skycoin/dmsg
# Visor and services over dmsghttp

Fixes [#skywire-services-481](https://github.com/SkycoinPro/skywire-services/issues/481)

 Changes:	
- Added sk flag for `dmsg-discovery`
- Added a way to serve `dmsg-discovery` via HTTP over dmsg
- Added `directClient`
- Added `http.Client` parameter to  `disc.NewHTTP()` 

How to test this PR:
Check [#skywire-services-486](https://github.com/SkycoinPro/skywire-services/pull/486)
